internal/mediators/config/registry.go

```diff
@@ -74,3 +84,15 @@ func GetDefault(opt Option) interface{} {
 	}
 	return opt.Default
 }
+
+// AllRegistered returns all registered options, excluding hidden ones
+func AllRegistered() []Option {
```

Comment:
```suggestion
// Registered returns all registered options, excluding hidden ones
func Registered() []Option {
```

"All" seems redundant.